documentation: Update README.md (#18)
tendermint/flutter: README.md

> # Flutter application for integrating with Cosmos-SDK

Reviewer: ```suggestion
# Flutter app for integrating with Cosmos SDK
```

> This is a sample application which is integrated to Cosmos-SDK for adding wallets through mnemonnics and sending tokens to different addresses.

Reviewer: ```suggestion
This is a sample application for a Cosmos SDK blockchain that supports basic wallet management (importing mnemonics), and sending tokens.
```

> ## Setup (Localhost)

Reviewer: ```suggestion
## Setup (on `localhost`)
```

- [Setup Flutter](https://flutter.dev/docs/get-started/install)
- [Install Starport](https://docs.starport.network/intro/install.html)
- Scaffold a blockchain using `starport app github.com/hello/planet`
- Navigate to `planet` folder and run `starport serve`

> - After it is successfully built, you will see two mnenomics for Alice and Bob along with their wallet addresses in the terminal

Reviewer: ```suggestion
- After it is successfully built, you will see two mnemonics for Alice and Bob along with their wallet addresses in the terminal
```

> - Run your Flutter app without any extra command line arguments and add both wallets in your app using the mnemonics from the terminal

Reviewer: How do I run a Flutter app?

Author: The Flutter setup link describes everything in detail regarding setting up and running Flutter app. I will add separate steps as well

> ## Setup (Hosted Testnet)

Reviewer: ```suggestion
## Setup (Cosmos Hub Testnet)
```
